roles/ceph-defaults/handlers/main.yml

```diff
@@ -24,7 +24,7 @@
     # We do not want to run these checks on initial deployment (`socket.rc == 0`)
     - mon_group_name in group_names
     - not containerized_deployment
-    - inventory_hostname == groups.get(mon_group_name) | last
+    - inventory_hostname in groups.get(mon_group_name)
```

Comment:
this line become redundant to L25

Reply:
actually, we can't remove this line, otherwise, because of `serial: 1` we won't skip this task on first nodes.